src/modules/AdvancedPaste/AdvancedPaste/Strings/en-us/Resources.resw

```diff
@@ -225,4 +225,7 @@
   <data name="TermsLink.Text" xml:space="preserve">
     <value>OpenAI Terms</value>
   </data>
+  <data name="OpenAIGpoDisabled" xml:space="preserve">
+    <value>To custom with AI disabled by your organization</value>
```

Comment:
```suggestion
    <value>To custom with AI is disabled by your organization</value>
```

Same for the "AI not enabled" string. And we should decide if "not enabled" or "disabled" I think.

Reply:
In one case, I guess it was not enabled by the user, since it's disabled by default. In the other case, it's disabled by the organization? Semantics, semantics, but I'm not sure either 🤷

Reply:
Good catch on this one. Committed it, thank you!

Reply:
> Good catch on this one. Committed it, thank you!

But we can fix the other one to `is not enabled` at least.